src/verinfast/agent.py

```diff
@@ -214,6 +216,7 @@ def upload(self, file: str, route: str, source: str = '', isJSON=True):
             return False
 
     def formatGitHash(self, hash: str):
+        hash = hash.replace("'","")
```

Comment:
Do we need to delete `'`? Should we also delete `"`? Not sure how they're getting into the git hash in the first place but want to fix it once.